apps/web/src/components/HintsAccordion/HintsAccordion.tsx

```diff
@@ -0,0 +1,96 @@
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Heading,
+} from "@chakra-ui/react";
+
+import { useColor } from "../../styles/useColor";
+import { type SignPage } from "../Titles";
+
+type HintData = {
+  header?: string;
+  description?: string;
+};
+
+export const Hints: Record<SignPage, HintData> = {
+  TezSignPage: {
+    header:
+      "A confirmation takes 1 block (10 seconds). The transaction becomes final and irreversible after 2 blocks (20 seconds).",
+    description:
+      "A confirmation means that the block is confirmed by the network but the operation can still be canceled in the unlikely case of a crypto attack. Finality means that the operation is irreversible.",
+  },
+  ContractCallSignPage: {
+    header:
+      "A confirmation takes 1 block (10 seconds). The transaction becomes final and irreversible after 2 blocks (20 seconds).",
+    description:
+      "A confirmation means that the block is confirmed by the network but the operation can still be canceled in the unlikely case of a crypto attack. Finality means that the operation is irreversible.",
+  },
+  DelegationSignPage: {
+    header: "It takes 4 cycles (~6 days) to start receiving delegation rewards.",
+    description:
+      "Delegation rewards are distributed every 2 cycles (~3 days). Ensure that the delegation fee offered by the baker is less than 100%; otherwise, you will not receive rewards. Delegation means that all your funds are delegated to one baker. The funds are still spendable, and you can cancel or change the delegation at any time.",
+  },
+  UndelegationSignPage: {
+    header:
+      "Immediately stops delegation and initiates unstaking. Restoring delegation rewards takes 4 cycles (~6 days). Unstaking takes 4 cycles (~10 days).",
+    description:
+      "Undelegation is applied immediately, so you will not receive rewards anymore. It takes 4 cycles (~6 days) to start receiving delegation rewards again after a new delegation. By undelegating, you also initiate the unstaking process if you have staked. Unstaking will take approximately 4 cycles (~10 days) to complete, after which your staked funds will be returned to your balance.",
+  },
```

Comment:
```suggestion
      "Undelegation is applied immediately, and the user will  stop accruing **staking** rewards. However, the user should receive delegation rewards for the current and the next 2 cycles, as the baking rights are computed 2 cycles in advance. By undelegating, you also initiate the unstaking process if you have staked. Unstaking will take approximately 4 cycles (~10 days) to complete, after which your staked funds  can be finalized in order to be made  ",
  },
```

This is my understanding, which is consistent with the assumption that unstaked tez would only count as after they are no longer slashable. Waiting for the answer from L1 team on this one.